lib/rubocop/cop/lint/eval.rb

```diff
@@ -8,9 +8,10 @@ class Eval < Cop
         MSG = 'The use of `eval` is a serious security risk.'
 
         def on_send(node)
-          receiver, method_name, = *node
+          receiver, method_name, arg = *node
```

Comment:
`*args` is not needed, because we only care about the first one. We are not checking anything in the optional 2nd, 3rd, and 4th args.